electron-app/src/server/websites/misskey/misskey.service.ts

```diff
@@ -131,13 +132,13 @@ export class MissKey extends Website {
 
     const instanceInfo: MissKeyInstanceInfo = this.getAccountInfo(data.part.accountId, INFO_KEY);
     const chunkCount = instanceInfo?.configuration?.statuses?.max_media_attachments ?? 4;
-    const maxChars = instanceInfo?.configuration?.statuses?.max_characters ?? 500;
+    this.MAX_CHARS = instanceInfo?.configuration?.statuses?.max_characters ?? 500;
```

Comment:
issue: not a safe use of a global still.

Reply:
Missed MissKey! Resolved.